packages/block-library/src/social-link/save.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+
+export default function save( { attributes, className } ) {
+	const { icon, url } = attributes;
+
+	if ( ! url ) {
+		return null;
+	}
+
+	const classes = classNames( 'wp-social-icon', `wp-social-icon-${ icon }` );
```

Comment:
Sometimes this is referred to as `social-links` and sometimes as `social-icon`.

Reply:
Fixed, standardized everything on social-link name